src/components/events/EventForm.jsx

```diff
@@ -683,10 +686,7 @@ function EventDatetimeInput({
               disabled={!startDateInput || disabled}
               minDateTime={
                 startDateInput
-                  ? (startDateInput instanceof Date && !isDayjs(startDateInput)
-                      ? dayjs(startDateInput)
-                      : startDateInput
-                    ).add(1, "minute")
+                  ? dayjs(startDateInput).add(1,"minute")
```

Comment:
I was having some issues with this thing, so make sure this change is working fine...

Reply:
Same, I tested it earlier multitple times this seems to have worked.